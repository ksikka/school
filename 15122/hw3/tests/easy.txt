aah ace
test case
compute program
bug cat
computer programs
slow fast
devil angel
