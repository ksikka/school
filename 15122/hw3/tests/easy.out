ace <-- ale <-- all <-- aal <-- aah

case <-- base <-- bast <-- best <-- test

No path found from compute to program

cat <-- bat <-- bag <-- bug

No path found from computer to programs

fast <-- fist <-- fiat <-- flat <-- flaw <-- flow <-- slow

angel <-- anger <-- auger <-- luger <-- leger <-- lever <-- level <-- devel <-- devil


takes 10 sec (BS - 5 sec)